vita3k/util/include/util/log.h

```diff
@@ -69,5 +71,3 @@ std::string log_hex(T val) {
     using unsigned_type = typename std::make_unsigned<T>::type;
     return fmt::format("0x{:0X}", static_cast<unsigned_type>(val));
 }
-
```

Comment:
keep new line in end file

Reply:
It has one now. It was previously missing.
A missing line at the end of a file has the sign ![sign](https://user-images.githubusercontent.com/25406440/107401375-19642700-6b03-11eb-9509-387e55b1ce4a.PNG) and I removed it.